ontopy/ontology.py

```diff
@@ -190,6 +190,46 @@ def __objclass__(self):
         # Play nice with inspect...
         pass
 
+    def __hash__(self):
+        """ Returns hash based on base_iri
+        This is done to keep Ontology hashable when defining __eq__.
+        """
+        return hash(self.base_iri)
+
+    def __eq__(self, other):
+        """Checks if this ontology is equal to other.
+
+        Equality of all triples obtained from self.get_triples(),
+        i.e. all triples except blank nodes.
+        """
+        return set(self.get_triples()) == set(other.get_triples())
+
+    def _sorted_entities(self):
+        """Return sorted entities
+        """
+        return sorted(e.iri for e in self.get_entities(
```

Comment:
```suggestion
        return sorted(_.iri for _ in self.get_entities(
```
A trick in Python is to use an underscore (`_`) for an irrelevant variable. Here the `e` variable is, first of all, a bad name for a variable (it's difficult to understand what is meant by "e"), but other than that, it's not used for anything other than being the iterable variable. Mainly for the latter reason, you can use `_` instead, as it's not a necessary variable for further use.